src/lib/MetalCam.mm

```diff
@@ -154,6 +147,13 @@ - (void) update {
     if(openglMode){
         [self _updateOpenGLTexture];
     }
+
+#if defined(__IPHONE_13_0) && defined(ARBodyTrackingBool_h)
```

Comment:
So I'm still seeing use of the `ARBodyTrackingBool_h` define here and there. Don't worry about that, I will go and clean that up a bit. 

For clarification, this can be written like this instead 
```javascript
#if defined(__IPHONE_13_0)
if(ARBodyTrackingConfiguration.isSupported && [_session.configuration isKindOfClass:[ARBodyTrackingConfiguration class]]){
        // update depth textures
        [self _updateMatteTextures: commandBuffer];
    }
#endif
```

The goal with my comment was to get rid of using preprocessor statements since there is a built-in way to check the current configuration which makes it less confusing for people to utilize different functionality.

Reply:
Ah crap, I realized I'm looking at an older commit. Assuming you changed this already.